src/graphql/status.rs

```diff
@@ -181,7 +181,7 @@ impl ConfigMutation {
 
         if *is_local {
             warn!("Config is local");
-            return Ok(false);
+            return Err("Config is local".to_string().into());
```

Comment:
The `Err` message is sufficiently clear to API users, but the warning message may not be as clear to those who see it among log messages. In fact, users attempting to set the config remotely is more significant than simply running with a local configuration. Since the API users themselves triggered this action, they don't need to be notified of it. However, those reviewing the log messages should be notified, as they may lack this context.

Reply:
Elaboration is added.